server/src/repositories/logger.repository.ts

```diff
@@ -1,10 +1,10 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, Scope } from '@nestjs/common';
 import { ClsService } from 'nestjs-cls';
 import { LogLevel } from 'src/entities/system-config.entity';
 import { ILoggerRepository } from 'src/interfaces/logger.interface';
 import { ImmichLogger } from 'src/utils/logger';
 
-@Injectable()
+@Injectable({ scope: Scope.TRANSIENT })
```

Comment:
This is necessary so each injected instance is independent and `.setContext` in one service doesn't override that of another